azurerm/resource_arm_function_app_test.go

```diff
@@ -0,0 +1,122 @@
+package azurerm
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func TestAccAzureRMFunctionApp_basic(t *testing.T) {
```

Comment:
can we add another acceptance test for going from `V1 -> Beta` (and back)? It's possible to use multiple test steps to do this and assert on the value of the `version` field in each step - as we do here: https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/azurerm/resource_arm_traffic_manager_endpoint_test.go#L111

Reply:
Sure thing.